src/promptflow/promptflow/azure/_storage/cosmosdb/span.py

```diff
@@ -23,8 +23,9 @@ class Span:
     resource: dict = None
     id: str = None
     partition_key: str = None
+    created_by: dict = None
```

Comment:
in which case user will query with tenant_id?

Reply:
Just a common practice for `created_by` info. Don't have very special reason for it.